src/game/Board.java

```diff
@@ -48,32 +48,30 @@ public Snake snake() {
 		return snake;
 	}
 
-	public Food food() {
+	public FoodGenerator food() {
```

Comment:
Here, food() method is actually a getter for FoodGenerator, so you should name this getFoodGenerator(). Same for snake().

Reply:
I changed names this methods

Reply:
"I changed this method's name."